tests/templates/kuttl/resources/01-install-opa.yaml.j2

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: opa.stackable.tech/v1alpha1
+kind: OpaCluster
+metadata:
+  name: opa
+spec:
+  version: {{ test_scenario['values']['opa'] }}
```

Comment:
What do you think about `opa-latest`? ;)

Reply:
OPA is so lightweight, i figured here it does not hurt to have tested all the versions. Although we are testing nothing in the OPA pods (rather if its set in the daemonset correctly), so i could do it. What do you think?

Reply:
OPA images are small agreed. On the other hand i would prefer consistency across the operators.
Feel free too leave it as is but than please put in a comment like "Testing all OPA versions here as the images and containers are very lightweight"

Reply:
done https://github.com/stackabletech/opa-operator/pull/347/commits/1c6f0593d758c7d3995b5391e1aaafd39dee235e